red/api/auth/index.js

```diff
@@ -40,8 +40,9 @@ function init(runtime) {
     settings = runtime.settings;
     log = runtime.log;
     if (settings.adminAuth) {
-        Users.init(settings.adminAuth);
-        Tokens.init(settings.adminAuth,runtime.storage);
+        var mergedAdminAuth = Object.assign(settings.adminAuth, settings.adminAuth.module);
```

Comment:
`Object.assign` modifies the target object in place and returns it. So this will be changing `settings.adminAuth`. I'm not sure we want to modify the settings object like that. I understand its a convenient way to use the `module` properties, but I think it needs to be done without `Object.assign`.

Reply:
Thank you for your feedback. I understood I have to fix my code not to change `settings.adminAuth`. In this case, I think adding an empty object(`{}`) as the first parameter of `Object.assign` can fix it.